force-app/main/default/lwc/accountModelSettings/accountModelSettings.html

```diff
@@ -7,12 +7,81 @@
                 onsettingssaving={handleSettingsSaving}
                 onsettingssavecompleted={handleSettingsSaveCompleted}
             >
-                <!-- Body of Account Model Settings page (populates Slot in child component) -->
-                <lightning-button
-                    label="Test Button"
-                    disabled={affordancesDisabled}
-                ></lightning-button>
             </c-settings-save-canvas>
+
+            <!-- Body of Account Model Settings page (populates Slot in child component) -->
+            <template if:true={accountModelSettingsViewModel}>
+                <div class="slds-grid slds-grid_vertical-align-start slds-wrap">
+                    <!-- Default Account Record Type Setting -->
+                    <div class="slds-col slds-size_2-of-3 slds-p-bottom_x-large">
+                        <div class="slds-text-title slds-text-color_default slds-p-left_small">
+                            {labelReference.stgAccModelTitle}
+                        </div>
+                        <div class="slds-text-body_small slds-text-color_weak slds-p-horizontal_large">
+                            {labelReference.stgHelpAccountModel}
+                        </div>
+                    </div>
+
+                    <div class="slds-col slds-size_1-of-3 slds-col_bump-right slds-p-right_x-large">
+                        <lightning-combobox
+                            name="defaultAccountModel"
+                            label="Default Account Record Type"
+                            value={accountModelSettingsViewModel.defaultAccountRecordType.value}
+                            placeholder=""
+                            options={accountModelSettingsViewModel.defaultAccountRecordType.options}
+                            onchange={handleDefaultAccountModelChange}
+                            disabled={affordancesDisabled}
+                            variant="label-hidden"
+                        ></lightning-combobox>
+                    </div>
+
+                    <!-- Administrative Account Record Type Setting -->
+                    <div class="slds-col slds-size_2-of-3 slds-p-bottom_x-large">
+                        <div class="slds-text-title slds-text-color_default slds-p-left_small">
+                            {labelReference.stgAdminAccountRecordType}
+                        </div>
+                        <div class="slds-text-body_small slds-text-color_weak slds-p-horizontal_large">
+                            {labelReference.stgHelpAdminRecType}
+                        </div>
+                    </div>
+
+                    <div class="slds-col slds-size_1-of-3 slds-col_bump-right slds-p-right_x-large">
+                        <lightning-combobox
+                            name="adminAccountModel"
+                            label="Administrative Account Record Type"
```

Comment:
Same here for the label. We could also consider using a labeled by for this, indicating the heading is the label.